### ISSUE TITLE
Won't open on Win11

### ISSUE THREAD

**Issue by Hipperooni** [andyvorld/LGSTrayBattery#23]
Windows 11 seems adamant to not run this, says it contains "unwanted software or virus". Other version of this didn't have this issue.

**Comment by andyvorld** [comment]
I am assuming this is in relation to the v2_alpha? An in other versions you refer to the previous non GHUB version?

This is weird as outside the code logic only the targeting .Net framework has changed from 4.6 to 4.7.
Are there anymore to the error? As I do not have access to a windows 11 machine at this time.

**Comment by Hipperooni** [comment]
Error message: https://gyazo.com/9ac073b5e3056a5f2559cde5499a01e6
I use the LGSTrayBattery_GHUB version currently and works fine except it doesn't autostart.

**Comment by andyvorld** [comment]
Yeah, now I am seeing it too, I had my entire build directory white listed so I didnt see any errors like that.
From what I can tell it is a false positive, I have checked it with [Virus total as well](https://www.virustotal.com/gui/file/37cd287b480b166bdc4e38e89c244712948b725b2f577c0a3855cfbe5f10a0cf?nocache=1).

I just tried rebuilding it and dropping 32bit support, and that seems to make my windows 10 defender happy.

**Comment by Hipperooni** [comment]
Hoorah! Now fixed. TYVM!
